test/ziggurat/middleware/default_test.clj

```diff
@@ -9,6 +9,29 @@
 (use-fixtures :once (join-fixtures [fix/mount-only-config
                                     fix/silence-logging]))
 
```

Comment:
We should also add the following two unit tests for the method `deserialize-message` :
1. should flatten the protobuf struct if `flatten-protobuf-struct?` is true
2. should not flatten the protobuf struct if `flatten-protobuf-struct?` is false

Reply:
Made requested changes